tests/test_flake8_comprehensions.py

```diff
@@ -18,7 +18,7 @@ def test_version(flake8dir):
 def test_C400_pass_1(flake8dir):
     flake8dir.make_example_py(
         """
-        foo = [x for x in range(10)]
+        foo = [x + 1 for x in range(10)]
```

Comment:
Note that many of the tests needed tweaking so as not to violate the new `C416` rule.